en/developer_guide/routing.rst

```diff
@@ -0,0 +1,79 @@
+Routing
+=======
+
+Base URL
+--------
+
+The page base url is compose with domain name and language prefix.
+To manage all possibilities, Open Orchestra associate each website in database with aliases that contain:
+
+* **domain**: domain name
+* **language**: language prefix
+* **scheme** : to choose with which protocol page can be access
+
+Here is an example of some base URL generated for a website associate to two domains ``dev.example.com`` and ``test.example.com`` with english and french languages and default languages english.
+
++----------+--------+------------------+----------+-----------------------------+
+| Alias Id | Scheme | Domain           | Language | Base URL                    |
```

Comment:
this array seems wrong to me, it should also specify which are the language prefix for each alias